skeleton.js

```diff
@@ -0,0 +1,16 @@
+var scraper = require('website-scraper');
+
+scraper.scrape({
+	urls: ['https://stat.qa.wordpress.boston.com/2016/04/04/stat-dataviz-skeleton-post/'],
+	directory: 'app/assets/',
+	subdirectories: [
+		{directory: 'img', extensions: ['.jpg', '.png', '.svg']},
+		{directory: 'js', extensions: ['.js']},
+		{directory: 'css', extensions: ['.css']}
+	 ],
+  sources: [
+		{selector: 'img', attr: 'src'},
+		{selector: 'link[rel="stylesheet"]', attr: 'href'},
+		{selector: 'script', attr: 'src'}
+	  ]
+}).then(console.log).catch(console.log);
```

Comment:
W/r/t to the subdirectories and sources config arrays above, can we copy the defaults from node_modules/website-scraper/lib/config/defaults.js since those cover more selectors and also covers putting font files in to subdirs. Once you copy these default config arrays to the script, please also add '.woff2' as a font extension so that those files get put in a 'vendor/stat/fonts' subdir as well. Feel free to make any other tweaks to the new/extended defaults as necessary as well.